app/src/rgb_underglow.c

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (c) 2020 Nick Winans <nick@winans.codes>
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <device.h>
+#include <init.h>
+#include <kernel.h>
+
+#include <math.h>
+#include <stdlib.h>
+
+#include <logging/log.h>
+
+#include <drivers/led_strip.h>
+#include <device.h>
+#include <drivers/spi.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+#define STRIP_LABEL		    DT_LABEL(DT_ALIAS(led_strip))
```

Comment:
Instead of an `alias`, we should be using a chosen node, e.g. `zmk,underglow = &led_strip` in the DT.